core/src/storage/sd_seed_backup.py

```diff
@@ -0,0 +1,57 @@
+from micropython import const
+from typing import TYPE_CHECKING
+
+from trezor import io, utils
+from trezor.sdcard import with_filesystem
+
+if TYPE_CHECKING:
+    from typing import TypeVar, Callable
+
+    T = TypeVar("T", bound=Callable)
+
+if utils.USE_SD_CARD:
+    fatfs = io.fatfs  # global_import_cache
+
+SD_CARD_HOT_SWAPPABLE = False
+SD_SALT_LEN_BYTES = const(512)
```

Comment:
please rename to something more sensible

Reply:
Done in 2e71232f57f6a1ac4119959677f12e193b4024d3